xml/System.Xml/XmlException.xml

```diff
@@ -347,8 +347,8 @@ The `XmlException` is a generic XML-related error that is thrown if problems occ
         <ReturnType>System.Int32</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the line number indicating where the error occurred.</summary>
-        <value>The line number indicating where the error occurred.</value>
+        <summary>Gets the zero-based line number indicating where the error occurred.</summary>
```

Comment:
```suggestion
        <summary>Gets the 1-based line number indicating where the error occurred.</summary>
```

Reply:
is `1-based` preferred over `one-based`?